rest/model/admin.go

```diff
@@ -687,6 +689,29 @@ func (a *APIAuthConfig) ToService() (interface{}, error) {
 	}, nil
 }
 
+type APIBucketConfig struct {
+	LogBucket     *string `json:"log_bucket"`
+	LogBucketType *string `json:"log_bucket_type"`
```

Comment:
do we need to pass through the bucket type? Is the type ever configured outside of testing?

Reply:
I think it is worth it for local development.